Formula/execline.rb

```diff
@@ -17,7 +17,6 @@ class Execline < Formula
   def install
     system "./configure",
       "--prefix=#{prefix}",
-      "--disable-static",
```

Comment:
Btw, there's always the alternative for patching the build to use dynamic libraries, with the mentioned caveat that the patch won't be accepted upstream.